tutorials/nservicebus-sagas/1-saga-basics/Solution/Billing/Program.cs

```diff
@@ -1,27 +1,39 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
 using NServiceBus;
 
-namespace Billing
+namespace Billing;
+
+class Program
 {
-    class Program
+    static async Task Main(string[] args)
     {
-        static async Task Main()
-        {
-            Console.Title = "Billing";
+        Console.Title = "Billing";
+
+        var builder = Host.CreateApplicationBuilder(args);
+
+        var endpointConfiguration = new EndpointConfiguration("Billing");
 
-            var endpointConfiguration = new EndpointConfiguration("Billing");
+        endpointConfiguration.UseSerialization<SystemJsonSerializer>();
 
-            var transport = endpointConfiguration.UseTransport<LearningTransport>();
+        endpointConfiguration.UseTransport<LearningTransport>();
 
-            endpointConfiguration.UseSerialization<SystemJsonSerializer>();
+        endpointConfiguration.SendFailedMessagesTo("error");
+        endpointConfiguration.AuditProcessedMessagesTo("audit");
 
-            var endpointInstance = await Endpoint.Start(endpointConfiguration);
+        // So that when we test recoverability, we don't have to wait so long
+        // for the failed message to be sent to the error queue
```

Comment:
```suggestion
        // Decrease the default delayed delivery interval so that we don't
        // have to wait too long for the a message to be moved to the error queue
```